sites/ademe/src/backend/templates/richie/base.html

```diff
@@ -3,6 +3,12 @@
 
 {% block body_footer_title %}{% endblock body_footer_title %}
 
+{% block topbar_contact %}
+<li class="topbar__item topbar__item--cta">
+    <a href="https://www.fun-services.help/hc/fr">{% trans "Aide" %}</a>
```

Comment:
I checked. In fact it is only generating a `.pot` file because there was no reexisting `.po` file. We didn't see it in git status because the `.pot` file is git ignored. I will now add it to your PR and so we can merge it.